src/views/TableDetail.vue

```diff
@@ -225,6 +250,10 @@ export default Vue.extend({
         type: 'all',
       });
     },
+
+    async updateTypes() {
+      this.columnTypes = await api.tableColumnTypes(this.workspace, this.table);
+    },
```

Comment:
If this function is called repeatedly just to update the types, shouldn't this just be as computed function? I'm guessing this is due to the call itself being async, but this could be another spot to use `asyncComputed`. Example use below

https://github.com/multinet-app/multinet-client/blob/1cb0b0e35f3a092d887b34afe2f74804c4a211b0/src/components/PermissionsDialog.vue#L295-L306

Reply:
I didn't realize we were already using asyncComputed. Have we run into any problems with it?

I'll reformulate this with asycComputed in the meantime.

Reply:
(or rather, instead of "didn't realize", I should have said "forgot" :face_with_head_bandage:)

Reply:
See c2eeec158ffab077cc8ee20171effb90a836c4f5.